cmd/init.go

```diff
@@ -36,6 +40,17 @@ func Init(c *git.Client, args []string) error {
 		return fmt.Errorf("Invalid init command. Must only provide one directory.")
 	}
 
+	if template != "" {
+		if !filepath.IsAbs(template) {
```

Comment:
I'm not sure I see why this needs to be made absolute, but I guess it doesn't hurt either.

Reply:
There is code deeper in the init command that is doing chdir and causing relative paths to not be found relative to where dgit was invoked. That's why the change to absolute paths is needed here.

Reply:
Makes sense (well, not really, but it's the chdir that doesn't make sense, your workaround in this PR is fine.. I'll have to look into it sometime.)